helmclient.go

```diff
@@ -57,6 +57,7 @@ type Config struct {
 
 	RestConfig      *rest.Config
 	TillerNamespace string
+	TillerImage     string
```

Comment:
Giving it a second thought we can even make it an internal. So it's only for changing during the test.

```suggestion
```